### PR TITLE
Merge 2.3 into master

### PR DESCRIPTION
**Pull request by Doodoune**

### CODE REVIEW

File: src/Oro/Bundle/PimFilterBundle/Filter/ProductValue/ChoiceFilter.php [hunk 1]
```diff
@@ -106,6 +106,7 @@ protected function getFormOptions()
     {
         $attribute = $this->getAttribute();
 
+        /* TODO @merge remove null condition on master, optionRepoClass to delete*/
```
Comment (grena) [inline src/Oro/Bundle/PimFilterBundle/Filter/ProductValue/ChoiceFilter.php]: Has it been done?

File: src/Akeneo/Pim/Enrichment/Component/Product/Validator/Constraints/FileValidator.php [hunk 2]
```diff
@@ -16,6 +16,9 @@
  */
 class FileValidator extends ConstraintValidator
 {
+    /** @var array|null */
```
Comment (pierallard) [inline src/Akeneo/Pim/Enrichment/Component/Product/Validator/Constraints/FileValidator.php]: Remove |null

File: src/Akeneo/Pim/Enrichment/Component/Product/Validator/Constraints/FileValidator.php [hunk 3]
```diff
@@ -24,6 +27,14 @@ class FileValidator extends ConstraintValidator
         BaseFileValidator::MIB_BYTES => 'MiB',
     ];
 
+    /**
+     * @param array|null $extensionToMimeTypeMapping
```
Comment (pierallard) [inline src/Akeneo/Pim/Enrichment/Component/Product/Validator/Constraints/FileValidator.php]: remove |null

File: src/Oro/Bundle/PimFilterBundle/Filter/ProductValue/ChoiceFilter.php (hunk 1 above)
Comment (pierallard) [reply src/Oro/Bundle/PimFilterBundle/Filter/ProductValue/ChoiceFilter.php]: Same question, seems to be done, remove the comment

File: src/Akeneo/Pim/Enrichment/Component/Product/Validator/Constraints/FileValidator.php (hunk 3 above)
Comment (Doodoune) [reply src/Akeneo/Pim/Enrichment/Component/Product/Validator/Constraints/FileValidator.php]: done

File: src/Akeneo/Pim/Enrichment/Component/Product/Validator/Constraints/FileValidator.php (hunk 2 above)
Comment (Doodoune) [reply src/Akeneo/Pim/Enrichment/Component/Product/Validator/Constraints/FileValidator.php]: done

File: src/Oro/Bundle/PimFilterBundle/Filter/ProductValue/ChoiceFilter.php (hunk 1 above)
Comment (Doodoune) [reply src/Oro/Bundle/PimFilterBundle/Filter/ProductValue/ChoiceFilter.php]: done

**Pull request merged**